### ISSUE TITLE
installer ignoring bin-dir config setting

### ISSUE THREAD

**Issue by bacheson** [jakoch/phantomjs-installer#16]
The installer script hard codes lower case "vendor" into the bin dir. The config setting should be used.

**Comment by jakoch** [comment]
Hey Blake!

Yes, this issue was reported before and is fixed in `2.0.0` and `dev-master`.
It's present in `1.9.8` and lower.
Please change your version constraint.
You need to require the latest version of "jakoch/phantomjs-installer" and add an version alias for the PhantomJS version: `"dev-master#099c14d as 1.9.8"`.

---

`composer.json`
```
{
    ,
    "config": {
        "bin-dir": "binaries",
        "vendor-dir": "libs"
    },
    "scripts": {
        "post-install-cmd": [
            "PhantomInstaller\\Installer::installPhantomJS"
        ],
        "post-update-cmd": [
            "PhantomInstaller\\Installer::installPhantomJS"
        ]
    }
}
```

**Comment by bacheson** [comment]
Unfortunately casperjs doesn't support phantoms 2.0. Thanks for the heads up!
